<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParseUtilPage</class>
 <widget class="QWidget" name="ParseUtilPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="parseLogFilesButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Start parsing debug files</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="parseLogFileResLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
